src/mixlib_install_updater.js

```diff
@@ -15,8 +15,8 @@ function checkForUpdates(currentVersion) {
   let _channel = workstation.getUpdateChannel();
   if (_platformInfo == null) {
     _platformInfo = workstation.getPlatformInfo();
-    
-    if (_platformInfo == null) {
+
+    if (_platformInfo == null || currentVersion == "development") {
```

Comment:
I see you already handled it :) 👍 I wonder what we could do to be able to show that there is an update in dev too?